src/Calculator/GraphingSettings.xaml

```diff
@@ -0,0 +1,136 @@
+                                            <Setter Target="AccessibilityBorder.BorderBrush" Value="{ThemeResource ControlPointerOverBorderBrush}" />
+                                            <Setter Target="AccessibilityBorder.Background" Value="{ThemeResource ControlSelectedBackgroundBrush}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                </VisualStateGroup>
+                                <VisualStateGroup x:Name="CheckStates">
+                                    <VisualState x:Name="Checked">
+                                        <VisualState.Setters>
+                                            <Setter Target="VisualBorder.BorderBrush" Value="{ThemeResource BackgroundColorBrush}" />
+                                            <Setter Target="VisualBorder.Background" Value="{ThemeResource BackgroundColorBrush}" />
+                                            <Setter Target="ContentPresenter.Background" Value="{ThemeResource BackgroundColorBrush}" />
+                                            <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource ForegroundColorBrush}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                    <VisualState x:Name="Unchecked" />
+                                    <VisualState x:Name="Indeterminate" />
+                                </VisualStateGroup>
+                            </VisualStateManager.VisualStateGroups>
+                        </Grid>
+                    </ControlTemplate>
+                </Setter.Value>
+            </Setter>
+        </Style>
+    </Page.Resources>
+
+    <Grid>
+        <StackPanel Margin="10,10,10,10">
+            <TextBlock x:Name="GridHeading" Text="Grid_d" FontSize="28"/>
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="30"/>
+                    <RowDefinition Height="40"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="30"/>
+                    <RowDefinition Height="40"/>
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="150"/>
+                    <ColumnDefinition Width="150"/>
+                    <ColumnDefinition Width="70"/>
+                </Grid.ColumnDefinitions>
+
+                <TextBlock Grid.Row="0" Grid.Column="0" x:Name="x_MinHeading" Text="x-min_d" VerticalAlignment="Center"/>
+                <TextBlock Grid.Row="0" Grid.Column="1" x:Name="x_MaxHeading" Text="x-max_d" VerticalAlignment="Center"/>
+                <TextBox Grid.Row="1" Grid.Column="0" x:Name="x_min" LosingFocus="OnLosingFocus" KeyDown="OnKeyDown" VerticalAlignment="Center"/>
```

Comment:
I think we should remain consistent with the casing and call this y_Min, same comment below for y_Max, etc.

Reply:
Done